### PR TITLE
Issue 69 into web-dev

### PR DESCRIPTION
**Pull request by dobber1611**

### CODE REVIEW

File: web/public/views/students.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+				<th>studentID</th>
+				<th>Student Name</th>
+				<th>Last Log Date</th>
+				<th>Action</th>
+			</tr>
+		</thead>
+		<tfoot>
+			<tr>
+				<th>studentID</th>
+        <th>Student Name</th>
```
Comment (sorianog) [inline web/public/views/students.html]: code formatting issue

File: web/public/js/services/StudentListService.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+angular.module("StudentListService", []).factory("students", function($log, $http) {
+	var vm = this;
```
Comment (sorianog) [inline web/public/js/services/StudentListService.js]: what does "vm" stand for?

File: web/public/js/appRoutes.js [hunk 3]
```diff
@@ -12,6 +12,11 @@ angular.module("appRoutes", []).config(
 			controller: "StatsController as vm"
 		})
 
+		.when("/students", {
+			templateUrl: "views/students.html",
+			controller: "StudentListController as vm"
```
Comment (sorianog) [inline web/public/js/appRoutes.js]: oh vm is studentlistcontroller. vm is a weird variable name to me.

Comment (dobber1611) [inline web/public/js/appRoutes.js]: Its a standard naming for angular.

File: web/public/views/students.html (hunk 1 above)
Comment (dobber1611) [inline web/public/views/students.html]: when I pulled the code it doesnt look like this.

**Pull request merged**